docs/readmes/proposals/p026_magma_inbound_roaming_extentions.md

```diff
@@ -0,0 +1,61 @@
+# Magma extensions for inbound roaming
+
+Author(s): [@arsenii-oganov]
+
+Last updated: 02/18/2022
+
```

Comment:
We should add the gdoc as reference here cc @uri200

Reply:
Can you add the google doc with the design here please